src/sentry/static/sentry/app/views/projectFilters.jsx

```diff
@@ -651,9 +662,9 @@ const ProjectFilters = React.createClass({
     let {formattedData} = this.state;
 
     return ReactDOMServer.renderToStaticMarkup(
-      <div style={{width: '150px'}}>
+      <div style={{width: '175px'}}>
         <div className="time-label"><span>{timeLabel}</span></div>
-        <div>{intcomma(totalY)} {totalY > 1 ? t('total events') : t('total event')}</div>
+        <div>{intcomma(totalY)} {totalY != 1 ? t('total events') : t('total event')}</div>
```

Comment:
alternatively we could use `tn` from `locale.jsx`:
```javascript
tn('total event', 'total events', totalY)
```

Reply:
I always forget about that